src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.cs

```diff
@@ -359,12 +359,18 @@ public static RenameDeclarationLocationReference[] CreateDeclarationLocationAnno
 
                 if (overriddenSymbol != null)
                 {
-                    overriddenSymbol = SymbolFinder.FindSourceDefinition(overriddenSymbol, solution, cancellationToken);
+                    // Unfortunately we cannot easily make CreateDeclarationLocationAnnotations async, as it's used in the rewriter that is rewriting trees.
```

Comment:
yeah.  it's a pity.  i would love if we moved off of a rewriter.  or, alternatively, do the computation up front of the edits to make, then have hte rerwriter just be a synchronous pass.  rewriters that do complex work inside of them break my brain.

Reply:
If I ever need to rewrite it, yes, this is getting rewritten to not use a rewriter... (hah!)